tests/blockchain/test_blockchain.py

```diff
@@ -1510,7 +1510,7 @@ async def test_bad_timestamp(self, bt, with_softfork2):
             # enable softfork2 at height 0, to make it apply to this test
             # the test constants set MAX_FUTURE_TIME to 10 days, restore it to
             # default for this test
-            constants = bt.constants.replace(SOFT_FORK2_HEIGHT=0, MAX_FUTURE_TIME=5 * 60)
+            constants = bt.constants.replace(SOFT_FORK2_HEIGHT=0, MAX_FUTURE_TIME=5 * 60, MAX_FUTURE_TIME2=2 * 60)
```

Comment:
this is already the default. I don't understand in what way this makes a difference

Reply:
@arvidn  its using the test constants which have a higher timestamp limit (10 days), I am setting it to the mainnet constants, because the test needs to use the real constants in order to test bad timestamps.